### PR TITLE
Migrate Code Base

### PR DESCRIPTION
**Pull request by zachcarp**
A decision was made to make this project fully open. This migrates code from an internal code base to public.

### CODE REVIEW

File: pipelines/winget-cli-restsource-ci-pr.yml [hunk 1]
```diff
@@ -1,9 +1,5 @@
 # Copyright (c) Microsoft Corporation. All rights reserved
-# CI/PR pipeline for winget-cli-restsource
-
-# Commit triggers
-trigger:
-- main
+# CI pipeline for OWCValidation
```
Comment (msftrubengu) [inline pipelines/winget-cli-restsource-ci-pr.yml:2]: >OWCValidation [](http://example.com/codeflow?start=18&length=13)

winget-cli-restsource

File: scripts/Release/managedsak_onetime_setup.ps1 [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+    [Parameter(Mandatory=$false)]
+    [string]
+    $regenerationPeriod = "P60D",
+
+    #The default here is for the msft tenant.
+    #For production/PME, pass in id: e6045dae-1e30-48d4-9502-f77712ec84c8
+    [Parameter(Mandatory=$false)]
+    [string]
+    $keyVaultObjectId = "93c27d83-f79b-4cb2-8dd4-4aa716542e74"
```
Comment (msftrubengu) [inline scripts/Release/managedsak_onetime_setup.ps1:40]: Is it ok to leave this in open source code?

**Comment by msftrubengu** [comment]
> <?xml version="1.0"?>

This file shouldnt be checked in. I deleted it in private repo

---
Refers to: src/documentation/Microsoft.WinGet.RestSource.Util.xml:1 in 3ed285c. [](commit_id = 3ed285c0de490a85f884f6ff90639404ba7f8ee6, deletion_comment = False)

File: scripts/Release/managedsak_onetime_setup.ps1 (hunk 2 above)
Comment (zachcarp) [reply scripts/Release/managedsak_onetime_setup.ps1:40]: Yes. This is the object id for the cross-tenant service principle that enables this feature.

**Pull request closed without merging**